visualizer/index.html

```diff
@@ -165,6 +166,50 @@
       color: white;
     }
 
+    #actions {
+      display: flex;
+      flex: 1;
+      position: fixed;
+      right: 20px;
+      margin-top: 40px;
+    }
+    #actions .actionColumn {
+      margin-left: 15px;
+    }
+    #actions .actionEntries {
+      flex-direction: column;
+      align-self: flex-start;
```

Comment:
I think you also might not need this. At least, when I disabled it, it didn't seem to change anything.